app.js

```diff
@@ -24,6 +26,7 @@ var seaTac = new Store('SeaTac', 6, 24, 1.2);
 var southCenter = new Store('South Center', 11, 38, 1.9);
 var bellevueSquare = new Store('Bellevue Square', 20, 48, 3.3);
 var alki = new Store('Alki', 3, 24, 2.6);
+
 //render is a property of the store constructor object, NOT each instance
 //1.run generate hourly for each store
 //2.get the table by id (done)
```

Comment:
Given that this is a global var for `tableEl` I would put that at the top of the file, so we can see that and know all of your globals.